src/pages/api/city.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import nextConnect from 'next-connect';
+
+import pool from 'utils/db';
+
+const handler = nextConnect().get(
+  (req: NextApiRequest, res: NextApiResponse) => {
+    pool
+      .query(`SELECT * FROM "City"`)
+      .then((result: any) => {
+        if (result.rowCount > 0)
+          return res.status(200).json({ success: true, City: result.rows });
+        return res
```

Comment:
It's not an error if there is no rows returning, you return an empty array and it's fine